frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -75,161 +86,128 @@ export function BillingV2({ redirectPath = '' }: BillingV2Props): JSX.Element {
                     You are currently on a free trial until <b>{billing.free_trial_until.format('LL')}</b>
                 </AlertMessage>
             ) : null}
-            {!billing && !billingLoading ? (
-                <div className="space-y-4">
-                    <AlertMessage type="error">
-                        There was an issue retrieving your current billing information. If this message persists please
-                        {supportLink}.
-                    </AlertMessage>
+            {!billing?.billing_period && (
+                <>
+                    <BillingHero />
```

Comment:
I reintroduced the colorful hero at the top

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/7335343/205432270-c0006255-297b-479c-9627-7e8d7237ad67.png">

Reply:
😅 I intentionally left it out because I think it could really use some love from a design standpoint. But if you really want it there that's fine.

Reply:
I know, but without the page looked too serious 😅